Test/datatypes/t4.bpl

```diff
@@ -0,0 +1,42 @@
+// RUN: %parallel-boogie "%s" > "%t"
```

Comment:
The names of the test files don't have any meaning, so they might as well be a single file.

Reply:
I understand. But separate files were helpful to me organizing the nature of different tests. So I would prefer to leave them separate. Would it be helpful if I added a comment in each file about the nature of the tests in it?

Reply:
Could you give the files more specific names?

Reply:
Done.